app-creator/code-editor/deserializer.js

```diff
@@ -281,6 +305,15 @@ function createApp(template) {
     return ui.Panel({ style: style, layout: layout });
   };
 
+  /**
+   * Helper function for creating sidemenu elements.
+   */
+  app.createSidemenuElement = function (obj, style) {
+    // Sidemenu is an object with the properties: sidePanel and contentPanel.
+    var mobileMenu = sidemenu.createSidemenu(style);
+    return mobileMenu.init();
```

Comment:
I think it make sense to consolidate this into one method, unless you plan on exposing more methods, to which I would recommend using a constructor pattern:
```
return sidemenu
  .createSidemenu(style)
  .addXFeature()
  .init();
```